### ISSUE TITLE
Archive inactive frameworks 

### ISSUE THREAD

**Issue by krausest** [krausest/js-framework-benchmark#1421]
According to the [deprecation policy](https://github.com/krausest/js-framework-benchmark/blob/master/README.md#history) I'd like to archive some inactive frameworks. Archived frameworks will appear in the history section in the README with a link to the last result table they were included it. The cleanup allows adding new implementations while keeping the benchmark duration bearable.

I'm informing the contributors in this issue. Please let me know if there are good reasons to keep the implementation.

The following candidates appear to have litte activity in the last year. Is it OK to archive them?

- [ ] vuerx-jsx @ryansolid
- [ ] ko-jsx @ryansolid
- [ ] mobx-jsx @ryansolid 
- [x] uhydro @Krutsch: done
- [x] domdiff @WebReflection  (I know feels like groundhog day, but maybe it turns out differently a year later): done
- [x] neverland @WebReflection: done
- [x] lighterhtml @WebReflection: done
- [x] heresy @WebReflection: done
- [x] hullo @arturkulig done
- [x] resonatejs @CodeMonkeyAlpha: done 
- [x] etch (repo is readonly): done 
- ~~s2 @gr0uch~~
- ~~redom @pakastin~~
- [x] endorphin @sergeche: done  
- [x] sifrr @aadityataparia: done
- [x] 1more @Freak613: done   
- ~~doohtml @hman61~~
- [x] sledgehammer (repo is readonly): done  
- [x] domvm: done
- [x] fullweb-helpers (repo is readonly): done 
- [x] fullweb-template (repo is readonly): done 

(legend: checked = will be archived, strike through = will be kept)
Non-keyed will follow...

**Comment by leeoniya** [comment]
you can archive domvm, both keyed and non-keyed. i dont actively maintain it and dont really plan to. ivi does what domvm set out to do, but does it better and much faster & smaller.

**Comment by pakastin** [comment]
RE:DOM is actively used in all my projects and others as well, it's just so minimalistic there's not constant need to push updates. I'd still hope you would keep it, i'm going to update at least the packaging/building part soon.

**Comment by CodeMonkeyAlpha** [comment]
Hi Stefan,

 

Hope you are well. Yes feel free to remove resonatejs if it is creating unnecessary work / noise. I have made a number of updates to it but haven’t published them, and frankly don’t have the time at the moment as it would require work on the documentation. It was designed specifically for one project / team. I wasn’t really under the illusion anyone but me and a few friends would be using it commercially. 

 

It was nice to see that it stacked up reasonably well against other complete frameworks.

 

Thanks

 

Terra

 

 

From: Stefan Krause ***@***.***> 
Sent: Sunday, October 15, 2023 9:07 PM
To: krausest/js-framework-benchmark ***@***.***>
Cc: CodeMonkeyAlpha ***@***.***>; Mention ***@***.***>
Subject: [krausest/js-framework-benchmark] Archive inactive frameworks (Issue #1421)

 

According to the deprecation policy <https://github.com/krausest/js-framework-benchmark/blob/master/README.md#history>  I'd like to archive some inactive frameworks. Archived frameworks will appear in the history section in the README with a link to the last result table they were included it. The cleanup allows adding new implementations while keeping the benchmark duration bearable.

I'm informing the contributors in this issue. Please let me know if there are good reasons to keep the implementation.

The following candidates appear to have litte activity in the last year. Is it OK to archive them?

*	[ ] vuerx-jsx @ryansolid <https://github.com/ryansolid> 
*	[ ] ko-jsx @ryansolid <https://github.com/ryansolid> 
*	[ ] mobx-jsx @ryansolid <https://github.com/ryansolid> 
*	[ ] uhydro @Krutsch <https://github.com/Krutsch> 
*	[ ] domdiff @WebReflection <https://github.com/WebReflection>  (I know feels like groundhog day, but maybe it turns out differently a year later)
*	[ ] neverland @WebReflection <https://github.com/WebReflection> 
*	[ ] lighterhtml @WebReflection <https://github.com/WebReflection> 
*	[ ] heresy @WebReflection <https://github.com/WebReflection> 
*	[ ] hullo @arturkulig <https://github.com/arturkulig> 
*	[ ] resonatejs @CodeMonkeyAlpha <https://github.com/CodeMonkeyAlpha> .
*	[ ] etch (repo is readonly)
*	[ ] s2 @gr0uch <https://github.com/gr0uch> 
*	[ ] redom @pakastin <https://github.com/pakastin> 
*	[ ] endorphin @sergeche <https://github.com/sergeche> 
*	[ ] sifrr @aadityataparia <https://github.com/aadityataparia> 
*	[ ] 1more @Freak613 <https://github.com/Freak613> 
*	[ ] doohtml @hman61 <https://github.com/hman61> 
*	[ ] sledgehammer (repo is readonly)

Non-keyed will follow...

**Comment by sergeche** [comment]
You can archive endorphin framework, working on a new version right now (maybe with different name), will add it via PR

**Comment by hman61** [comment]
Please DON'T remove *doohtml* or *doohtml*-dom, I'm planning a minor
update in the next 2 weeks.

- Henrik

**Comment by gr0uch** [comment]
[s2](https://github.com/gr0uch/s2) is being actively maintained and used in production. i may have forgot to link back to the repo from here.

**Comment by WebReflection** [comment]
I’m OK if you remove the libraries you’ve mentioned as I’m not working much on these  👍

**Comment by birkskyum** [comment]
Seems like "[fullweb-helpers-v0.1.0](https://github.com/fullwebdev/fullwebdev#readme)" has been archived too.

**Comment by krausest** [comment]
Ping @ryansolid: Should we archive vuerx-jsx, ko-jsx, mobx-jsx?

**Comment by krausest** [comment]
Ping @arturkulig: Can we archive hullo?

**Comment by krausest** [comment]
Ping @aadityataparia: Can we archive sifrr?

**Comment by arturkulig** [comment]
> Ping @arturkulig: Can we archive hullo?

Yes, that's what I understood was about to happen [previously](https://github.com/krausest/js-framework-benchmark/issues/1146#issuecomment-1328274125).

**Comment by aadityataparia** [comment]
archiving sifrr is fine by me

**Comment by krausest** [comment]
Last ping for @ryansolid to stop me from removing ko-jsx, vuerx-jsx, mobx-jsx.

**Comment by birkskyum** [comment]
Maybe vuerx-jsx (no-virtual-dom POC) can stay until https://github.com/vuejs/core-vapor (official no-virtual-dom) is out?

**Comment by Erithax** [comment]
[Dojo](https://github.com/dojo/framework) has been inactive for 2 years

**Comment by krausest** [comment]
I'm closing the issue due to inactivity.
